muxnetprofile/main.c

```diff
@@ -464,8 +465,6 @@ void ui_refresh_task() {
         if (lv_group_get_obj_count(ui_group) > 0) adjust_wallpaper_element(ui_group, 0);
         adjust_panel_priority(ui_mux_panels, sizeof(ui_mux_panels) / sizeof(ui_mux_panels[0]));
 
-        lv_obj_move_foreground(overlay_image);
```

Comment:
Please place this and the next line back

Reply:
all of the cleanup requests are complete. Testing note: you need the change I posted a PR for in muos/internals along with this PR's changes to muxnetwork and muxnetprofile's main.c files for empty wifi passwords to work.